here-naksha-lib-psql/src/commonTest/kotlin/naksha/psql/UpdateFeatureTest.kt

```diff
@@ -46,17 +46,17 @@ class UpdateFeatureTest : PgTestBase(NakshaCollection("update_feature_test_c"))
         executeWrite(updateFeaturesReq)
 
         // And: Retrieving feature by id
-        val retrievedFeature = executeRead(ReadFeatures().apply {
+        val retrievedTuples = executeRead(ReadFeatures().apply {
             collectionIds += collection.id
             featureIds += initialFeature.id
-        }).let { response ->
-            val responseFeatures = response.features
-            assertEquals(1, responseFeatures.size)
-            responseFeatures[0]!!
-        }
+            queryHistory = true
+        }).tuples
+
+        val retrievedUpdatedTuple = retrievedTuples.first { it?.tuple?.meta?.action() == Action.UPDATED }!!
+        val retrievedHstCreatedTuple = retrievedTuples.first { it?.tuple?.meta?.action() == Action.CREATED }!!
```

Comment:
I was wondering that if this is done then we could test the scenario where w examine not only versions abut also content of hsitory feature after updating (I would probably do it in separate test)